docs/source/faq/faq.md

```diff
@@ -16,15 +16,18 @@ Kedro was originally designed by [Aris Valtazanos](https://github.com/arisvqb) a
 Currently, the core Kedro team consists of
 [Ahdra Merali](https://github.com/AhdraMeraliQB),
 Andrew Mackay,
```

Comment:
I think Mackay's GH is https://github.com/Mackay031, unless there's a reason he doesn't want this here?

Reply:
Good shout, I've added it!